### PR TITLE
Update mobilenetv2 symbol definition

### PR DESCRIPTION
**Pull request by liangfu**
The pretrained model is provided in [this](https://github.com/liangfu/mxnet-mobilenet-v2) repository, where both model size and inference accuracy have been reproduced.

### CODE REVIEW

File: example/image-classification/symbols/mobilenetv2.py [hunk 1]
```diff
@@ -17,60 +17,239 @@
...
-    multiplier : float, default 1.0
-        The width multiplier for controling the model size. The actual number of channels
-        is equal to the original channel size multiplied by this multiplier.
-    ctx : Context, default CPU
-        The context in which to initialize the model weights.
-    """
-    net = MobileNetV2(multiplier=multiplier, classes=num_classes, **kwargs)
-    net.initialize(ctx=ctx, init=mx.init.Xavier())
-    net.hybridize()
+def invresi_blocks(data, in_c, t, c, n, s, prefix):
```
Comment (zhreshold) [inline example/image-classification/symbols/mobilenetv2.py]: function name?

File: example/image-classification/symbols/mobilenetv2.py [hunk 2]
```diff
@@ -17,60 +17,239 @@
...
+            (6, 320, 1, 1), # 160x7x7 -> 320x7x7
+        ],
+        'filter_num_before_gp': 1280, # 320x7x7 -> 1280x7x7
+    } 
+}
 
-    # plot network graph
-    mx.viz.plot_network(sym, shape={'data': (8, 3, 224, 224)},
-                        node_attrs={'shape': 'oval', 'fixedsize': 'fasl==false'}).view()
+class MNetV2Gen(object):
```
Comment (zhreshold) [inline example/image-classification/symbols/mobilenetv2.py]: use MobileNetv2

**Comment by liangfu** [comment]
changes to class and function names have been made according to the suggestions.

File: example/image-classification/symbols/mobilenetv2.py [hunk 3]
```diff
@@ -17,60 +17,202 @@
...
-    # plot network graph
-    mx.viz.plot_network(sym, shape={'data': (8, 3, 224, 224)},
-                        node_attrs={'shape': 'oval', 'fixedsize': 'fasl==false'}).view()
+class MobileNetV2(object):
+    def __init__(self, data_wh, multiplier, **kargs):
+        super(MobileNetV2, self).__init__()
+        self.data_wh=data_wh
+        self.multiplier=multiplier
+        if self.data_wh in MNETV2_CONFIGS_MAP:
+            self.MNetConfigs=MNETV2_CONFIGS_MAP[self.data_wh]
```
Comment (zhreshold) [inline example/image-classification/symbols/mobilenetv2.py]: mixing Camel case with under score in names are not pretty, even though it is in example folder.

File: example/image-classification/symbols/mobilenetv2.py [hunk 4]
```diff
@@ -17,60 +17,202 @@
...
+        else:
+            self.MNetConfigs=MNETV2_CONFIGS_MAP[(224, 224)]
+    
+    def genNet(self, class_num=1000, **configs):
```
Comment (zhreshold) [inline example/image-classification/symbols/mobilenetv2.py]: and function name shouldn't have capitals according to PEP8

**Comment by hetong007** [comment]
LGTM

**Comment by liangfu** [comment]
add a link to pretrained model ?

**Pull request merged**